src/global.ts

```diff
@@ -338,6 +338,14 @@ export interface GetTokenSilentlyOptions {
    */
   timeoutInSeconds?: number;
 
+  /**
+   * If true, the full response from the /oauth/token endpoint (or the cache, if the cache was used) is returned
+   * (minus `refresh_token` if one was issued). Otherwise, just the access token is returned.
+   *
+   * The default is `false`.
+   */
+  verboseResponse?: boolean;
```

Comment:
Not married to the name, if there are better suggestions.

Reply:
I have no better naming, but I do feel like verbose has a `debugging/error` connotation to it. But that might just be because I only ever use that term when configuring log levels for errors.

Reply:
That's fair, how about `detailedResponse`?

Reply:
I believe that should work better as opposed to verbose, ye. `detailedResponse` or `fullResponse`.

Reply:
Renamed to `detailedResponse`.